dragonflybsd/README.md

```diff
@@ -68,14 +68,14 @@ After installing DragonflyBSD we can run QEMU without mounted `.iso`:
 As a default, we do not have terminal emulation. Only one output is on the
 graphic window with rendered console. To make development and validation easier,
 it is good to obtain an SSH connection between host and QEMU machine. To do
-that, you should set these values in `/etc/rc.conf` (on host machine):
+that, you should set these values in `/etc/rc.conf` (in QEMU VM):
 
 ```
  sshd_enable="YES"
  ifconfig_em0="DHCP"
 ```
 
-Now you need to send your public key to `/home/root/.ssh/authorized_keys` file
+Now, you need to send your public key to `/home/root/.ssh/authorized_keys` file
```

Comment:
@kotylamichal I do not get it. Send via SCP if SSH is not available? How?

Also: `IP of host machine in local network>` - do you mean `host` or maybe `VM` machine (the same error as above)? As I suspect the `/etc/rc.conf` was meant to be edited in QEMU VM, not in my host Linux distro.